### DISCUSSION TITLE
Neper Polycrystal - View setting

### DISCUSSION THREAD

**Discussion by kshankar7** [neperfepx/neper#803] [Q&A]
I need help in setting the right orientation for my viewport in a polycrystal visualization. 
Here is the output of my neper -V call
`neper -V al1_20k_gran.sim -step 42 -dataeltcol stress_eq -showcsys 1 -datacsyslabel X:Y:Z -datacsyscoo -0.1:-0.25:-0.1 -dataelt3dedgerad 0 -print test`
![test](https://github.com/neperfepx/neper/assets/86091354/7e662b22-0f9a-46c9-931f-d2ba935c87cd)
![test-scale3d](https://github.com/neperfepx/neper/assets/86091354/51658698-df81-4c05-866e-02b5777e74bf)

The view shows the polycrystal with "Z" up. I would like to rotate the model with "Y" up. Can someone help?

**Comment by rquey** [comment]
See [-camerasky](https://neper.info/doc/neper_v.html#cmdoption-camerasky), and you'll have to play with [-cameracoo](https://neper.info/doc/neper_v.html#cmdoption-cameracoo) too.
 ```
neper -T -n 10
neper -V n10-id1.tess -showcsys 1 -datacsyslength 1.3 -datacelltrs 0.5 -cameracoo 5:5:5 -camerasky 0:0:1 -print z
neper -V n10-id1.tess -showcsys 1 -datacsyslength 1.3 -datacelltrs 0.5 -cameracoo 5:5:5 -camerasky 0:1:0 -print y
```
![z](https://github.com/neperfepx/neper/assets/2659376/d40516ea-f4c6-4cae-9ae6-cfa4400cdd99)
![y](https://github.com/neperfepx/neper/assets/2659376/ec8ed14e-8c2c-4689-8e87-6364c77bddb2)
